config/extend.go

```diff
@@ -217,3 +217,32 @@ func parseMapExtend(loader *pkgload.PackageLoader, c *Converter, fullMethod stri
 		EmptySource: true,
 	})
 }
+
+func parseInitExtend(loader *pkgload.PackageLoader, c *Converter, fullMethod string) (*method.Definition, error) {
```

Comment:
We should reuse parseMapExtend here, no need for the duplication, we can support everything mapExtend does.